dev/TeachingTip/TeachingTipAutomationPeer.h

```diff
@@ -12,6 +12,22 @@ class TeachingTipAutomationPeer :
     // IAutomationPeerOverrides 
     winrt::AutomationControlType GetAutomationControlTypeCore();
     winrt::hstring GetClassNameCore();
+
+    winrt::WindowInteractionState InteractionState();
+    bool IsModal();
+    bool IsTopmost();
+    bool Maximizable();
+    bool Minimizable();
+    winrt::WindowVisualState VisualState();
+    void Close();
+    void SetVisualState(winrt::WindowVisualState state);
+    bool WaitForInputIdle(int32_t milliseconds);
+
+    void RaiseWindowClosedEvent();
+    void RaiseWindowOpenedEvent(wstring_view displayString);
```

Comment:
Pass struct types by const&
```suggestion
    void RaiseWindowOpenedEvent(wstring_view const& displayString);
```